cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -31,7 +33,40 @@ def __init__(self, client: CogniteClient, dataset_id: int, cogshop_version: str
         self._dataset_id = dataset_id
         self.cogshop_version = cogshop_version
 
-    def trigger_case(self, case: SHOPCase, source: str | None = None) -> SHOPRun:
+    def trigger_case(self, case: Case, concurrent_calls: int = 5) -> list[SHOPRun]:
```

Comment:
```suggestion
    def trigger_case(self, case: Case) -> list[SHOPRun]:
```
Lets hide this away in a config in the client, this means that you set it in the `__init__` of this class and not allow the user to set it directly.

```python
def __init__(...):
    self._CONCURRENT_CALLS = 5
```